host-interaction/process/inject/use-pe-injection.yml

```diff
@@ -0,0 +1,29 @@
+rule:
+  meta:
+    name: use pe injection
+    namespace: host-interaction/process/inject
+    author: 0x534a@mailbox.org
+    scope: function
+    att&ck:
+      - Defense Evasion::Process Injection::Portable Executable Injection [T1055.002]
+    examples:
+      - ce8d7590182db2e51372a4a04d6a0927a65b2640739f9ec01cfd6c143b1110da:0x4014E0
+    references:
+      - https://www.elastic.co/de/blog/ten-process-injection-techniques-technical-survey-common-and-trending-process
+  features:
+    - and:
+      - characteristic: loop
+      - optional:
+        - or:
+          - api: kernel32.OpenProcess
+          - api: ntdll.NtOpenProcess
+          - api: ntdll.ZwOpenProcess
+          - match: create process
+      - match: allocate RWX memory
+      - basic block:
+        - description: virtual address offset calculation
+        - and:
+          - mnemonic: and
+          - number: 0x0FFF
```

Comment:
I'm not sure if this is a key part in every implementation of `pe injection`

Reply:
I am also unsure regarding this block of the rule. I simply followed https://www.elastic.co/de/blog/ten-process-injection-techniques-technical-survey-common-and-trending-process in this case in order to write the rule. 

I think, you are the more proficient reverse engineers than me and maybe have more experience regarding pe injection. Do have certain samples in mind that implement pe injection in another way?

I will also do some research on my own but this will take some time.

Reply:
I say we remove this part.

Reply:
I think, we could then remove the whole rule since there is no real difference to `inject-thread.yml`.

Reply:
Ah, I see. Let's keep the rule as is then.